base_report_docx/report/report_docx.py

```diff
@@ -64,11 +64,19 @@ def create_source_docx(self, cr, uid, ids, context=None):
 
         report = self._get_convert_file(
             tmp_folder_name, output_report[output_type])
-
+        # Put the pdf to sharefolder before delete the temp folder
+        self.save_pdf_to_share(cr, uid, ids, tmp_folder_name, context)
         self._delete_temp_folder(tmp_folder_name)
 
         return (report, output_type)
 
+    def save_pdf_to_share(cr, uid, ids, tmp_folder_name, context):
```

Comment:
why not directly add here the function. it could be useful for other cases, no?

Reply:
No, other report not have the same logic. can't put it here.

Reply:
@elicoidal I've talked to @Reinhard-sheng and explain that would need about 8h to refactor this and the others functions. So feels like he can't be on time.

@Reinhard-sheng I think you can go ahead and next time just give us feedback if you have any concern.

cc @leo8tony